app/models/settings.ts

```diff
@@ -1,8 +1,8 @@
-export interface Settings {
-    homePage: "account" | "home" | "job";
+export enum Theme {
```

Comment:
While this looks nice to have in the settings, is it worth exposing if we only have 1??